config.yaml

```diff
@@ -92,3 +92,14 @@ options:
     type: int
     default: 10000
     description: Start the exporter at "exporter-port".
+  backup-basedir:
+    type: string
+    default: "/opt/backups"
```

Comment:
wouldn't be better to have the default as `/home/ubuntu` ?

Reply:
Updated to the default value used by the DB charms